cmd/utils/flags.go

```diff
@@ -241,6 +241,10 @@ var (
 		Usage: "Public address for block mining BLS signatures (default = first account created)",
 		Value: "0",
 	}
+	BLSwalletFlag = cli.BoolFlag{
+		Name:  "blswallet",
+		Usage: "Use hardware wallet for BLS signing. Overrides blsbase",
```

Comment:
nit: Remove "Overrides blsbase" and adjust description for blsbase to "Public address for block mining BLS signatures (default = first account created). Should not be set when using hardware wallet for BLS signing."